src/main/java/pl/pandziak/ticketbookingapp/domain/screening/ScreeningByTitleAndTimeComparator.java

```diff
@@ -0,0 +1,17 @@
+package pl.pandziak.ticketbookingapp.domain.screening;
+
+import java.util.Comparator;
+
+final class ScreeningByTitleAndTimeComparator implements Comparator<Screening> {
+
+    @Override
+    public int compare(Screening s1, Screening s2) {
+        int movieTitleCompare = s1.getMovie().compareToIgnoreCase(s2.getMovie());
```

Comment:
Czy to zadziała poprawnie dla polskich liter?

Reply:
Faktycznie nie działa dla polskich - nie dotestowałam tego przypadku. Można by użyć chociażby klasy java.text.Collator razem z java.util.Comparator, aby komparator poradził sobie z polskimi znakami. Tak już działa. :)